src/olympia/devhub/tasks.py

```diff
@@ -110,9 +110,9 @@ def create_version_for_upload(addon, upload,
         # review since listed and sideload aren't supported for creation yet.
         if addon.status == amo.STATUS_NULL:
             if disallow_preliminary_review:
-                addon.update(status=amo.STATUS_PUBLIC)
+                addon.update(status=amo.STATUS_NOMINATED)
             else:
-                addon.update(status=amo.STATUS_LITE)
+                addon.update(status=amo.STATUS_PENDING)
```

Comment:
Isn't `STATUS_PENDING` for personas ? Any anway, wouldn't you want `STATUS_LITE` in that case, like it was before ? Wait, scratch that, why isn't it `STATUS_UNREVIEWED` ?